pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -108,22 +110,24 @@ func (ed *eventDispatcher) dispatchEvent(pod *api.Pod, cgroupPath string, kind l
 
 		glog.Infof("Dispatching to event handler: %s", name)
 		reply, err := client.Notify(ctx, ev)
+		replies = append(replies, reply)
 		if err != nil {
-			return err
+			return replies, err
```

Comment:
If we just aggregate errors like replies then we can continue to isolate with the other delegates if one fails instead of doing this early return.